src/main/java/dan200/computercraft/shared/peripheral/speaker/SoundTask.java

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
You'd have to make them `final`. For `volume` and `pitch` you can always create `final` copies of the original variable.